android/expoview/src/main/java/host/exp/exponent/notifications/insecurecheduler/InsecureScheduler.java

```diff
@@ -0,0 +1,16 @@
+package host.exp.exponent.notifications.insecurecheduler;
+
+import android.content.Context;
+import android.os.Bundle;
+
+import java.util.HashMap;
+
+public interface InsecureScheduler {
```

Comment:
Then we could also move it to where other schedulers are — `schedulers` package, decreasing perceived complexity. 🙂

Reply:
InsecureScheduler is not safe cause it's not resilient to reboot(also timezone change and other problems).

Reply:
I undestand why it's "insecure", I just mean that in this case this doesn't concern the native developer. Its features are exposed to JS and it belongs logically to `schedules` package.

If we were to somehow make it obvious that this scheduler is somehow different than others I think a better way would be to name it `LegacyTimeoutScheduler`, not `InsecureScheduler`, since this way we will communicate its purpose better (if someone would have to debug a problem with scheduling notifications with timeout one would first look for `timeout` keyword among schedulers, not for `HereBeBugsScheduler` 😅 )